.github/workflows/default.yml

```diff
@@ -50,7 +50,15 @@ jobs:
       matrix:
         os: [ ubuntu-latest, windows-latest ]
         gradle: [ 7.0.1, current, rc ]
-        java: [ 8, 11, 16 ]
+        java: [ 8, 11, 17 ]
```

Comment:
Do we also need to update the release.yml workflow to use Java 17?

Reply:
Good catch 👍 I clearly missed that!